src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,114 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String LOGIN = "srgch7";
+    private static final String LOGIN_SECOND_OCCUR = "tnhxeR";
+    private static final String PASSWORD = "468271";
+    private static final int AGE = 18;
+    private static RegistrationServiceImpl registrationServiceImpl;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationServiceImpl = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user = createNewUser(LOGIN, PASSWORD, AGE);
+    }
+
+    @Test
+    void register_singleUser_ok() {
+        User registeredUser = registrationServiceImpl.register(user);
+        assertNotNull(registeredUser);
+        assertEquals(user.getLogin(), registeredUser.getLogin());
+    }
+
+    @Test
+    void registered_twoDifferentUsers_Ok() {
```

Comment:
```suggestion
    void register_twoDifferentUsers_Ok() {
```

Reply:
I changed the method name from this `registered_twoDifferentUsers_Ok` to that `register_twoDifferentUsers_Ok`.